h/api/search.py

```diff
@@ -31,7 +34,41 @@ def _match_clause_for_uri(uristr):
     }
 
 
-def build_query(request_params):
+def scan(es_client, query, fields):
```

Comment:
I'm trying to encapsulate elasticsearch in this module. So this module calls elasticsearch, other modules call this module. I know we said in a call recently that encapsulating Es in one place was desirable but not if there was a good reason for some other code to access it directly. When I wrote this code though our rule was simpler: encapsulate it here.

So I guess the options would be:

* worker.py imports elasticsearch directly and calls scan and bulk itself, but I consider accessing it directly a -1 and not sure there's a compelling enough reason to do it

* `from elasticsearch import bulk` in `search.py` instead of a wrapper function, but then you're gonna get unused import warnings from prospector